src/helpers/redux/reducers/index.ts

```diff
@@ -8,6 +9,7 @@ import { storiesReducer } from './stories';
 import { userReducer } from './user';
 
 export default combineReducers<IRootState>({
+    offline,
```

Comment:
wouldn't `offline` be part of the `app` reducer? Asking this in terms of organization :thinking:

Reply:
We can move when all SAGAS are implemented.